blockchain/forkid/forkid_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2025 The Kaia Authors
+// This file is part of the Kaia library.
+//
+// The Kaia library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The Kaia library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the Kaia library. If not, see <http://www.gnu.org/licenses/>.
+
+package forkid
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/kaiachain/kaia/common"
+	"github.com/kaiachain/kaia/params"
+)
+
+// TestCreation tests that different genesis and fork rule combinations result in
+// the correct fork ID.
+func TestCreation(t *testing.T) {
+	type testcase struct {
+		head uint64
+		want ID
+	}
+	tests := []struct {
+		config  *params.ChainConfig
+		genesis common.Hash
+		cases   []testcase
+	}{
+		// Mainnet test cases
+		{
+			params.MainnetChainConfig,
+			params.MainnetGenesisHash,
+			[]testcase{
+				{0, ID{Hash: checksumToBytes(0xdd58eb45), Next: 86816005}},          // Unsynced
+				{86816004, ID{Hash: checksumToBytes(0xdd58eb45), Next: 86816005}},   // Last Genesis block
+				{86816005, ID{Hash: checksumToBytes(0x7b1131bb), Next: 99841497}},   // First Istanbul block
```

Comment:
```suggestion
				{86816005, ID{Hash: checksumToBytes(0x7b1131bb), Next: 99841497}},   // First Istanbul+London+EthTxType block
```
Let's be more clear about it, since ForkID allows multiple forks at the same block. Please annotate other lines in this file, too.

Reply:
Fixed it in [82eb9d4](https://github.com/kaiachain/kaia/pull/588/commits/82eb9d4035809ad9d6be18cac768fa13f484596a).